packages/@react-stately/checkbox/test/useCheckboxGroupState.test.tsx

```diff
@@ -31,11 +32,22 @@ describe('useCheckboxGroupState', () => {
     render(<Test />);
   });
 
-  it('should return the same `name` that has been provided', () => {
+  it('should return the same `isDisabled` that has been provided', () => {
     function Test() {
-      const state = useCheckboxGroupState({name: 'foo'});
+      const state = useCheckboxGroupState({isDisabled: true});
 
-      expect(state.name).toBe('foo');
+      expect(state.isDisabled).toBe(true);
+
+      return null;
+    }
+    render(<Test />);
+  });
+
+  it('should return the same `isReadOnly` that has been provided', () => {
+    function Test() {
+      const state = useCheckboxGroupState({isReadOnly: true});
+
+      expect(state.isReadOnly).toBe(true);
 
       return null;
     }
```

Comment:
would be great to add a simple test for `isSelected` functionality as well

Reply:
I guess. We tend not to do too much unit testing of the stately part by itself given that it's already covered by the ARIA/Spectrum tests indirectly. This tends to lead to more maintainable tests. But I suppose since the others are already tested I can add a quick one.